src/components/HTMLEngineProvider/HTMLRenderers/MentionHereRenderer.tsx

```diff
@@ -1,26 +1,30 @@
 import React from 'react';
+import type {TextStyle} from 'react-native';
+import {StyleSheet} from 'react-native';
+import type {CustomRendererProps, TText} from 'react-native-render-html';
 import {TNodeChildrenRenderer} from 'react-native-render-html';
-import _ from 'underscore';
 import Text from '@components/Text';
 import useStyleUtils from '@hooks/useStyleUtils';
-import htmlRendererPropTypes from './htmlRendererPropTypes';
 
-function MentionHereRenderer(props) {
+function MentionHereRenderer({style, tnode}: CustomRendererProps<TText>) {
```

Comment:
To get rid of assertion in `index.ts` file
```suggestion
function MentionHereRenderer({style, tnode}: CustomRendererProps<TText | TPhrasing>) {
```